uwrt_utils/include/uwrt_utils/uwrt_can.h

```diff
@@ -0,0 +1,48 @@
+#include <string>
+#include<vector>
+#include<map>
+#include <thread>
+#include <mutex>
+#include <chrono>
+#include <linux/can.h>
+#include <linux/can/raw.h>
+#include <net/if.h>
+
+namespace uwrt_can {
+
+class UWRTCANWrapper {
+ public:
+  explicit UWRTCANWrapper(const std::string& name, const std::string& device, int delay_time = 1) 
+    : name_(std::move(name)),
+      device_(std::move(device)),
+      delay_time_(std::chrono::milliseconds(delay_time)) {};
+  ~UWRTCANWrapper() { read_thread_running_ = false; }
+
+  bool init(const std::vector<canid_t>& ids);
+
+  bool read(struct can_frame *frame, canid_t id);
+  bool write(const struct can_frame *frame);
```

Comment:
there should probably also be a write_and_wait_for_ack_frame type function that runs on a seperate sockethandle. for oneshot type commands, the fw is gonna send back an frame with some value just to inidicate acknowledgement of receiving and execution